### PR TITLE
Speech in noise

### PR DESCRIPTION
**Pull request by rinama**

### CODE REVIEW

File: src/cljs/audyx/test/vocal.cljs [hunk 1]
```diff
@@ -50,8 +51,32 @@
...
+
+(defn ^:export playNoise [full-yaml file-name volume]
+  (let [yaml-clj (js->clj full-yaml :keywordize-keys true)
+        {:keys [type fixed-signal]} (:loop yaml-clj)
+        volume-settings {:type type
+                         :fixed-signal-volume (:volume fixed-signal)
+                         :fixed-signal-type (:type fixed-signal)}]
+    (go->promise play-noise file-name yaml-clj volume volume-settings)))
+
+(defn ^:export getNoiseFile [file-name]
```
Comment (tzipisheinin) [inline src/cljs/audyx/test/vocal.cljs:71]: I think it is a wrong name `get` it's for retirning any data

File: src/cljs/audyx/test/vocal.cljs [hunk 2]
```diff
@@ -50,8 +51,32 @@
...
+  (go
+    (match (<! (audio-noise/create-noise-node :noise file-name))
+            [:error data] [:error data]
+            [:ok _] [:ok true])))
+
 (defn ^:export playSound [yaml transducer volume vocal-data vocal-index sequence-name cb]
-  (let [{:keys [type fixed-signal] :as yaml-clj} (js->clj yaml :keywordize-keys true)
+  (let [yaml-clj (js->clj yaml :keywordize-keys true)
+        {:keys [type fixed-signal]} (:loop yaml-clj)
```
Comment (tzipisheinin) [inline src/cljs/audyx/test/vocal.cljs:79]: Did you check it is inside loop?

File: src/cljs/audyx/audio_engine/vocals.cljs [hunk 3]
```diff
@@ -292,4 +292,4 @@
   (go->promise load-vocals file-name))
 
 (defn ^:export convertDbSplToDbFs [volume transducer]
-  (dbspl->dbfs volume transducer))
+  (dbspl->dbfs volume transducer))
```
Comment (tzipisheinin) [inline src/cljs/audyx/audio_engine/vocals.cljs:295]: revert

File: src/cljs/audyx/audio/ambiences/noise.cljs [hunk 4]
```diff
@@ -112,9 +112,10 @@
 
 (deftry create-noise-node [noise-type noise-path]
   (swap! noise-graph assoc :status :loading)
-  (go 
+  (go
     (match (log (<! (get-noise-data noise-type noise-path)))
-           [:ok [buffer yaml-data]] (do (swap! noise-graph assoc :buffer buffer)
+           [:ok [buffer yaml-data]] (do yaml-data
```
Comment (tzipisheinin) [inline src/cljs/audyx/audio/ambiences/noise.cljs:117]: Is it by mistake?

File: src/cljs/audyx/test/vocal.cljs [hunk 5]
```diff
@@ -67,4 +92,11 @@
                                              (recur))
                             :end-playing (cb "stop")))
           [:error error] (cb error)
-          :else nil))))
\ No newline at end of file
+          :else nil))))
+
```
Comment (tzipisheinin) [inline src/cljs/audyx/test/vocal.cljs:96]: remove line

File: src/cljs/audyx/test/vocal.cljs (hunk 2 above)
Comment (rivkykot) [reply src/cljs/audyx/test/vocal.cljs:79]: this data is in the loop,
before this development this data was not usage

File: src/cljs/audyx/audio/ambiences/noise.cljs (hunk 4 above)
Comment (rivkykot) [reply src/cljs/audyx/audio/ambiences/noise.cljs:117]: no, it is looks that the do get the `yaml-data`
and break line after `do`

**Pull request merged**